docs/source/blogs/release-0-3-0-blog/release-0-3-0-blog.rst

```diff
@@ -0,0 +1,169 @@
+
+As an example, the ``normalize_variant`` function which was used within a SQL expression above, can also be written as a Python function as follows:
+
+.. code-block::
+
+  from glow.functions import normalize_variant
+  function_normalized_variants_df = original_variants_df.withColumn( \
+    "normalizationResult", \
+    normalize_variant("contigName", "start", "end", "referenceAllele", "alternateAlleles", "/mnt/dbnucleus/dbgenomics/grch38/data/GRCh38_full_analysis_set_plus_decoy_hla.fa") \
+  )
+
+The same can be done in Scala assuming ``original_variant_df`` is defined in Scala:
+
+.. code-block:: scala
+
+  import io.projectglow.functions.normalize_variant
+  import org.apache.spark.sql.functions.col
+  val function_normalized_variants_df = original_variants_df.withColumn(
+    "normalizationResult",
+    normalize_variant(col("contigName"), col("start"), col("end"), col("referenceAllele"), col("alternateAlleles"), "/mnt/dbnucleus/dbgenomics/grch38/data/GRCh38_full_analysis_set_plus_decoy_hla.fa")
+  )
+
+Parsing of Annotation Fields
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In this release, the VCF reader and pipe transformer received equipment to parse ``ANN`` and ``CSQ`` ``INFO`` fields resulting form annotation tools such as `SnpEff <http://snpeff.sourceforge.net/index.html>`_ and  `VEP <https://www.ensembl.org/info/docs/tools/vep/index.html>`_ tools into a structured format instead of putting them as flat strings in the ``INFO`` field. This makes queries on annotations much simpler to write and much faster. An example of such a query in shown below with its result displayed in :numref:`figcsqdf`:
+
+.. code-block::
+
+  from pyspark.sql.functions import expr
+  variants_df = spark.read\
+    .format("vcf")\
+    .load("dbfs:/databricks-datasets/genomics/vcfs/loftee.vcf")
+  annotated_variants_df = original_variants_df.withColumn("Exploded_INFO_CSQ", expr("explode(INFO_CSQ)")).selectExpr("contigName", "start", "end", "referenceAllele", "alternateAlleles", "expand_struct(Exploded_INFO_CSQ)", "genotypes")
+
+
+.. figure:: figcsqdf.png
+   :align: center
+   :width: 800
+   :name: figcsqdf
+
+   The annotated DataFrame ``annotated_variants_df`` with expanded subfields of the exploded ``INFO_CSQ``
+
+Other Improvements
+~~~~~~~~~~~~~~~~~~
+This release also brought about improved implementations of Glow's linear and logistic regression functions resulting in ~50% improvement in their performance. The manner in which these functions are used remained unchanged and can be found in their documentations at :ref:`linear-regression` and :ref:`logistic-regression`, respectively.
```

Comment:
Glow 0.3.0 also includes optimized implementations of the linear and logistic regression functions, resulting in ~50% performance improvements. See the documentation at :ref:`linear-regression` and :ref:`logistic-regression`.